components/aws/sagemaker/tests/integration_tests/resources/config/xgboost-mnist-trainingjob-debugger/config.yaml

```diff
@@ -0,0 +1,59 @@
+PipelineDefinition: resources/definition/training_pipeline.py
+TestName: xgboost-mnist-trainingjob-debugger
+Timeout: 86400
+ExpectedTrainingImage: ((XGBOOST_REGISTRY)).dkr.ecr.((REGION)).amazonaws.com/sagemaker-xgboost:1.0-1-cpu-py3
+Arguments:
+  region: ((REGION))
+  image: ((XGBOOST_REGISTRY)).dkr.ecr.((REGION)).amazonaws.com/sagemaker-xgboost:1.0-1-cpu-py3
+  training_input_mode: File
+  hyperparameters:
+    max_depth: "5"
+    eta: "0.2"
+    gamma: "4"
+    min_child_weight: "6"
+    silent: "0"
+    subsample: "0.7"
+    num_round: "51"
+  channels:
+    - ChannelName: train
+      CompressionType: None
+      ContentType: text/csv
+      DataSource:
+        S3DataSource:
+          S3DataType: S3Prefix
+          S3DataDistributionType: FullyReplicated
+          S3Uri: s3://((DATA_BUCKET))/mnist_kmeans_example/input/valid_data.csv
+  instance_type: ml.m4.xlarge
+  instance_count: 1
+  volume_size: 5
+  max_run_time: 3600
+  model_artifact_path: s3://((DATA_BUCKET))/xgboost-debugger/output
+  network_isolation: "True"
+  traffic_encryption: "False"
+  spot_instance: "False"
+  max_wait_time: 3600
+  checkpoint_config: "{}"
```

Comment:
do you need to set spot and network related parameters ? how about using/setting default value in pipeline definition?